src/app.js

```diff
@@ -8,11 +8,13 @@ const Discord = require("discord.js");
 const presence = require("./functions/presence");
 const greetUser = require("./functions/greetUser");
 const serverCount = require("./functions/serverCount");
+const execute = require("./functions/execute");
 
 if (
     !process.env.BOT_TOKEN ||
     !process.env.BOT_ID ||
-    !process.env.DISCORD_BOATS_token
+    !process.env.DISCORD_BOATS_token ||
+    !process.env.MONGODB_URL
```

Comment:
You don't actually have to do this if else ladder. Code gets unnecessarily long because of this. Simply write:
`require('dotenv').config()`

Reply:
I need because heroku doesn't create .env file automatically

Reply:
But if I need to run it locally or allow other folks to run we need these.

Reply:
But if I need to run it locally or allow other folks to run we need these.

Reply:
> I need because heroku doesn't create .env file automatically

Heroku kind of ignores that statement. You don't need `if` statement for this.

Reply:
> > I need because heroku doesn't create .env file automatically
> 
> Heroku (applies to all hosting platforms) handles .env automatically. You don't need `if` statement for this. There will be a time when you have many env variables. You don't have to put all in `if` condition!